lang/java/reef-common/src/main/java/org/apache/reef/io/WorkingDirectoryTempFileCreator.java

```diff
@@ -19,59 +19,50 @@
 package org.apache.reef.io;
 
 import org.apache.reef.annotations.Provided;
+import org.apache.reef.io.parameters.TempFileRootFolder;
+import org.apache.reef.tang.Injector;
+import org.apache.reef.tang.JavaConfigurationBuilder;
+import org.apache.reef.tang.Tang;
+import org.apache.reef.tang.exceptions.InjectionException;
 
 import javax.inject.Inject;
 import java.io.File;
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
 import java.nio.file.attribute.FileAttribute;
-import java.util.logging.Level;
 import java.util.logging.Logger;
 
 /**
- * Creates temp files in a directory named "temp" within  the current working directory.
+ * Creates temp files in a directory named "temp" within the current working directory.
  */
 @Provided
 public final class WorkingDirectoryTempFileCreator implements TempFileCreator {
```

Comment:
Wouldn't this class go back to the version in `master` instead? We'd just need to bind `TempFileCreator` to the new implementation `ConfigurableDirectoryTempFileCreator` for the offending tests. Or am I missing the obvious?